service/worker/archiver/util.go

```diff
@@ -96,6 +97,17 @@ func shouldRun(probability float64) bool {
 	return rand.Intn(int(1.0/probability)) == 0
 }
 
+func isHistoryMutated(historyBlob *HistoryBlob, request *ArchiveRequest) bool {
```

Comment:
calling this historyMutated is more standard go: https://golang.org/doc/effective_go.html#Getters